lnrpc/rpc.proto

```diff
@@ -2794,12 +2794,40 @@ message ListPaymentsRequest {
     that pending payments, as well as failed payments will show up if this
```

Comment:
This commit should be squashed with the previous one so that each commit builds :)

Reply:
Have squashed those commits.